lib/formatters.js

```diff
@@ -138,70 +139,9 @@ function formatJsonTrace(level, context, message, args, err) {
 
   log.time = new Date();
   log.lvl = level;
-  log.err = serializeErr(err, level);
+  log.err = err && serializeErr(err).toObject(API.stacktracesWith.indexOf(level) > -1);
```

Comment:
Opinion: I tend to favor readability over lines of code.

Something like this
```
var logStackTrace = API.stacktracesWith.indexOf(level) > -1;
log.err = err && serializeErr(err).toObject(logStackTrace);
```

over
```
log.err = err && serializeErr(err).toObject(API.stacktracesWith.indexOf(level) > -1);
```